index.html

```diff
@@ -787,6 +787,20 @@ <h2>
             <td>
 Identifies a certain version timestamp of a <a>DID document</a> to be resolved
 (i.e., the <a>DID document</a> that was valid for a <a>DID</a> at a certain time).
+Note: This parameter may not be supported by all <a>DID methods</a>.
+            </td>
+          </tr>
+          <tr>
+            <td>
+<code>initial-values</code>
```

Comment:
I don't understand why this is plural, does it support some sort of array semantics in the value property? If not, change to "initial-value".